packages/dapp-svelte-wallet/api/src/lib-wallet.js

```diff
@@ -859,6 +859,10 @@ export function makeWallet({
         Object.fromEntries(
           Object.entries(amountKeywordRecord).map(
             ([keyword, { pursePetname, value }]) => {
+              // Automatically convert numbers to BigInts.
+              if (typeof value === 'number') {
+                value = BigInt(value);
```

Comment:
Actually, please do:
```js
import { Nat } from '@agoric/nat';
```

and:
```suggestion
                value = Nat(value);
```
since that protects against any loss of precision in the conversion.

Reply:
Done, thanks!